### DISCUSSION TITLE
reveal.js presentation: Slides with no title appear in the toc

### DISCUSSION THREAD

**Discussion by acmuehlhausen-smc** [quarto-dev/quarto-cli#3668] [Q&A]
Hey there,

I've got a small but annoying problem designing a reveal.js-presentation in Quarto: when I have slides with no title (e.g.: a contact slide at the end, a slide with a full-size image, a quote ...) it appears in the toc as an empty bullet point, like this:

**TOC**
- Fancy first slide
- On the essence of unicorns
- 

How can I get rid of that bullet point? I already tried [this](https://mine-cetinkaya-rundel.github.io/quarto-tip-a-day/posts/23-titleless-slides/) but it only worked for the hamburger menu, not the toc. Some googling showed me a [%notitle] param for reveal.js but I did not find a Quarto implementation for it yet.

Thanks for the help!
Alex

**Comment by cscheid** [comment]
Thanks! Tracking on #3671.
